packages/appstore/src/components/modals/cashier-wallet-modal/cashier-wallet-modal.scss

```
@@ -0,0 +1,177 @@
+.dc-modal__container_cashier-wallet-modal {
+    position: fixed;
+    inset: 0;
+    min-height: calc(100vh - 84px) !important;
+    min-width: 100vw !important;
+    margin: 48px 0 36px;
+    border-radius: unset;
+    background-color: var(--general-main-1);
+
+    @include mobile {
+        min-height: 100vh !important;
+        min-width: 100vw !important;
+        margin: 0;
+        z-index: 100;
+    }
+
+    .cashier-wallet-modal {
+        &__header {
+            display: flex;
+            position: relative;
+            height: 16rem;
+            padding: 2.4rem 4rem 7.2rem;
+
+            @include mobile {
+                height: 12.2rem;
+                padding: 1.6rem 1.6rem 5.6rem;
+            }
+
+            &--hide-title {
+                height: 9.8rem;
+                align-items: center;
+                justify-content: space-between;
+            }
+
+            &__demo--light:before {
```

better to define this color in theme.scss and remove this separation for dark and light.

```suggestion
            &__demo:before {
                  background-color: var(--new-color)
            }
```